CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Acra ChangeLog
 
+## [0.96.0](https://github.com/cossacklabs/acra/releases/tag/0.96.0), September 9th 2024
+
+_Core_:
+
+- **AcraServer**:
+  - Fixed panic with simultaneous signals processing [#707](https://github.com/cossacklabs/acra/pull/707)
+  - Improved encryptor config parsing logic [#676](https://github.com/cossacklabs/acra/pull/676), [#715](https://github.com/cossacklabs/acra/pull/665)
+  - Implemented MySQL PreparedStatements from SQL support [#665](https://github.com/cossacklabs/acra/pull/665)
```

Comment:
not clear what means `sql support`. Maybe let's rephrase to `Added support of Prepared statements through Text protocol in MySQL`?